### PR TITLE
Implement tests and add styles

### PR DESCRIPTION
**Pull request by danielmatama**
I completed the webpages
I implemented tests for the app.
I added cool styles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,192 @@
...
+- **[Home Page Displaying All Coins]**
+- **[Details Page Displaying Details Of a Selected Coin]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo and Video Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://danielmatama.github.io/metrics-app/)
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please fix the live demo deploy link, the current link is broken

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @danielmatama , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  border: 2px solid #d35400;
+  border-radius: 8px;
+}
+
+@media (max-width: 600px) {
+  .data-cont {
+    width: calc(100vw - 40px);
+    margin: 0 auto;
+  }
+}
```
Comment (Gambit142) [inline src/App.css:51]: - [OPTIONAL] This is the way your application looks on both clickable and non-clickable screens for mobile view

 
image

image

#### Kindly notice the white spaces on both screenshots. Ideally, the application should occupy the full width and height on any screen the user uses to view the application. I would advise you to work on the UI and ensure the white spaces are not visible as they are not part of the design. Kindly take a cue from the design template provided for the capstone:

image

#### Notice the way the application looks on the mobile phone in the screenshot above. 😎 💻

File: README.md [hunk 3]
```diff
@@ -1 +1,192 @@
-# metrics-app
\ No newline at end of file
+<a name="Metrics WebApp"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```
Comment (Gambit142) [inline README.md:7]: - [OPTIONAL] Kindly remove the Microverse logo. If you have your logo, you can replace the Microverse logo with it, but if you don't, kindly remove it. 👍

**Review by Gambit142 — APPROVED** [review]
Hi @danielmatama 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- The application has both the Home and details page ✔️ 
- Use of `react-router-dom` ✔️ 
- Use of duck pattern to structure the project's directory ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**